src/static-files/cs/dummy.html

```diff
@@ -0,0 +1,5 @@
+<div id="faq-like">
```

Comment:
When is this `dummy.html` loaded? Is there logic which show this page if there is no translation?

Reply:
I did it based on> https://github.com/ufal/clarin-dspace/blob/d20cbdb420e7c7ac01bfea8e3285b4ffaff1c1ed/dspace-xmlui/src/main/webapp/themes/UFAL/lib/html/cs/dummy.html#L3